### PR TITLE
Fonts: I created a fonts branch for managing all of the fonts files needed to mimic Hipcamp

### PR DESCRIPTION
**Pull request by hannahmanfredi**
I'd like to merge this hc-awesome font file for Calibre onto main so I can use it on my front-end.

### CODE REVIEW

File: exampleData.js
Comment (kimberlyjk) [inline exampleData.js:1]: this is smart! I might have to follow suite

File: questions.js
Comment (kimberlyjk) [inline questions.js]: you might want to put this file in your gitignore file

File: server/server.js
Comment (kimberlyjk) [inline server/server.js]: consider sending back a response to the site/app so that its not just waiting for a response but it mknows there was an error somewhere.

**Review by kimberlyjk — APPROVED** [review]
Nice progress! 

It looks like you may have two styles.css files. I would advise to make sure that you have one that has all the styling you need before merging these files

File: questions.js
Comment (hannahmanfredi) [reply questions.js]: oops lol ty

**Pull request closed without merging**
